### PR TITLE
Update documentation for Go-Guru switch

### PR DESCRIPTION
**Pull request by jredville**
This branch is based off of @edrex's work on switching the go layer to use go-guru (#6774). It updates the documentation to warn about Oracle's deprecation and a few usage notes on go-guru.

### CODE REVIEW

**Comment by jredville** [comment]
@bogdanteleaga any feedback? Trying to get @edrex's work moved forward

File: layers/+lang/go/README.org [hunk 1]
```diff
@@ -97,6 +98,18 @@ function testing to work.
 the coverage buffer. See [[https://www.gnu.org/software/emacs/manual/html_node/elisp/Choosing-Window.html][=display-buffer=]] for a list of possible functions.
 The default value is =display-buffer-reuse-window=.
 
+** Guru
+
+Go Oracle has been deprecated as of October 1, 2016, it's replacement is =go-guru=.
+
+If you would like to use the =goguru= bindings in your work, you will need to
+install it, and in your project you will need to set the scope with =SPC m f o=.
```
Comment (d12frosted) [inline layers/+lang/go/README.org]: Keybindings should be surrounded with `~`.

**Comment by jredville** [comment]
Updated, and will avoid in the future

**Comment by JAremko** [comment]
@d12frosted  @TheBB  :shipit:

**Comment by TheBB** [comment]
Thanks! Squashed your two commits and cherry-picked in develop. You can safely delete your branch.

**Pull request closed without merging**